### DISCUSSION TITLE
4 PCBs V54 R1.35 4€ each plus shipping. Optionally with SMDs soldered

### DISCUSSION THREAD

**Discussion by microtronics** [nliaudat/esp32_8ch_motor_shield#28] [Buy or Sell]
I have 4 PCBs left and sell them for the same price i paid. 4€ plus shipping from Germany. ROHS compliant HASL.
If someone really has issues with soldering the SMD components, I can do it at additional cost.
Remember to also fix: https://github.com/nliaudat/esp32_8ch_motor_shield/issues/27

**Comment by ewertmaster** [comment]
Hi, I am interested in the 4 PCBs. I also have no issues with soldering SMD parts. Did you eventually also have the Parts which you can sell? 
Alex

**Comment by microtronics** [comment]
Hi!
I can offer: 
4 PCBs 4€ each -> 16€
4 * 8 L9110S -> 10€
4 * 2 ShiftReg -> 4€
I probably do not have enough resistors lying around to populate all boards. I do have plenty of 10k or 100k 0805 to use as pullups, if you don't mind soldering smaller resistors to the pads. (0805 can be soldered to 1206 without issues, I have done that).
64 0805 100k -> 2€
For current sensing it is better to use 1206 as they can handle more power. I don't have enough of these and you'd have to pick the value depending on the motors and if your ESP32s are capable of measuring between 0 and ~100 mV as some of them are not.
For the capacitors, I didn't populate them as they add more issues than they solve.

**Reply by ewertmaster** [reply]
Okay, you can just put together a package of what you have and the rest I will get myself. Tell me what you want for it and how we can proceed.

**Reply by microtronics** [reply]
Total would be 32€ plus shipping, depending on where you live and the shipping method. You can also come and get the boards and parts if you live near Stuttgart.
You can send me a message here: -> contact established, address removed <-
You may know the forum. Best if you add your e-Mail there so I can contact you directly.

**Reply by ewertmaster** [reply]
Okay, I would contact you via mikrokontroller.net
Thank you
